### PR TITLE
General overhaul of FAQ

### PR DESCRIPTION
**Pull request by timsneath**
Changes proposed in this pull request:

* Overhauled and updated many sections of the FAQ

### CODE REVIEW

File: src/docs/resources/faq.md [hunk 1]
```diff
@@ -881,10 +832,9 @@ There's an API to find the list of licenses you need to show:
 
 ### Who works on Flutter?
 
-Flutter is an open source project. Currently, the bulk of the
-development is done by engineers at Google. If you're excited
-about Flutter, we encourage you to join the community and
-contribute to Flutter!
+Flutter is an open source project. Currently, the bulk of the development is
+done by engineers at Google. If you're excited about Flutter, we encourage you
+to join the community and contribute to Flutter!
```
Comment (sfshaza2) [inline src/docs/resources/faq.md:837]: =>
```
[contribute to Flutter][]!

[contribute to Flutter]: {{site.github}}/flutter/flutter/blob/master/CONTRIBUTING.md
```

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
A few things. Overall, a big improvement.

File: src/docs/resources/faq.md [hunk 2]
```diff
@@ -1,176 +1,144 @@
...
-apps and reduces the cost and complexity
-of app production across platforms.
-
-For designers, Flutter helps deliver the original
-design vision, without loss of fidelity or compromises.
-It also acts as a productive prototyping tool.
+Flutter is Google's portable UI toolkit for crafting beautiful, natively
+compiled applications for mobile, web, and desktop from a single codebase.
+Flutter works with existing code, is used by developers and organizations around
+the world, and is free and open source.
```
[the remark this answers is not in the document]
Comment (timsneath) [reply src/docs/resources/faq.md:13]: :) I just word wrapped it here... can you hear Martin saying these words though?

File: src/docs/resources/faq.md [hunk 3]
```diff
@@ -1,176 +1,144 @@
...
+For developers, Flutter lowers the bar to entry for building apps. It speeds app
+development and reduces the cost and complexity of app production across
+platforms.
 
-Fundamentally, Flutter is for users that want beautiful apps,
-with delightful motion and animation, and UIs with character
-and an identity all their own.
+For designers, Flutter provides a canvas for high-end user experiences. Fast
+Company described Flutter as [one of the top design ideas of the decade][] for
+its ability to turn concepts to be turned into production code without the
```
[the remark this answers is not in the document]
Comment (timsneath) [reply src/docs/resources/faq.md:25]: Good catch!

File: src/docs/resources/faq.md [hunk 4]
```diff
@@ -1,176 +1,144 @@
...
+We have seen people with very little programming experience learn and use
+Flutter for prototyping and app development.
 
 ### What kinds of apps can I build with Flutter?
 
-Flutter is optimized for 2D mobile apps that want to run on
-both Android and iOS. Flutter is also great for interactive
-apps that you want to run on your web pages or on the desktop.
-(Note that web support is in beta, and desktop support is in alpha.)
+Flutter is designed to support for 2D mobile apps that run on both Android and
```
[the remark this answers is not in the document]
Comment (timsneath) [reply src/docs/resources/faq.md:45]: What _was_ I thinking? Thanks :)

File: src/docs/resources/faq.md [hunk 5]
```diff
@@ -1,176 +1,144 @@
...
+of the many editors that support [editing Dart][].
 
 ### Does Flutter come with a framework?
 
-Yes! Flutter ships with a modern framework,
-inspired by React. Flutter's framework is designed
-to be layered and customizable (and optional).
-Developers can choose to use only parts of the
-framework, or a different framework.
+Yes! Flutter ships with a modern reactive framework. Flutter's framework is
```
[the remark this answers is not in the document]
Comment (timsneath) [reply src/docs/resources/faq.md:117]: Fixed

File: src/docs/resources/faq.md [hunk 6]
```diff
@@ -281,121 +243,108 @@ to expose the new mobile OS capability.
...
+produce a final binary for the App Store containing the latest compiler
+optimizations and features. The profile and debug frameworks contain only a
+_bitcode marker_, and are more representative of the engine's actual binary
+size. Whether you ship with bitcode or not, the increased size of the release
+framework is stripped out during the final steps of the build. These steps
+happen after archiving your app and shipping it to the store.
 
-Of course, we recommend that you measure your own app.
-To do that, see [Measuring your app's size][].
+Of course, we recommend that you measure your own app. To do that, see
```
[the remark this answers is not in the document]
Comment (timsneath) [reply src/docs/resources/faq.md:346]: I'm sorry! I use a fantastic VSCode extension called Rewrap, which just takes care of wrapping. It's configured (by default) to wrap on 80 columns.

**Comment by timsneath** [comment]
OK, updated. Thanks for the great review, @sfshaza2!

**Review by sfshaza2 — APPROVED** [review]
Thanks for improving this! LGTM

**Pull request merged**